### DISCUSSION TITLE
How do i go back to stock firmware.

### DISCUSSION THREAD

**Discussion by Linusisnice** [keirf/flashfloppy#842] [Q&A]
i bought a gotec drive with flashfloppy but it seems to be configured for amiga. I cant get the ff.cfg so I want to know how do I revert back to the stock firmware?

**Comment by keirf** [comment]
FlashFloppy firmware is universal. What system are you wanting to use it with?

You can't revert to factory firmware as it is read protected and noone except factory has a copy of it.
